### PR TITLE
v0.25.0

### PR DESCRIPTION
**Pull request by 1ec5**
Updated the Xcode project, CocoaPods podspecs, and readme for v0.25.0.

/cc @mapbox/navigation-ios

### CODE REVIEW

File: TestHelper/Info.plist [hunk 1]
```diff
@@ -15,8 +15,8 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.25.0</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>2</string>
```
Comment (vincethecoder) [inline TestHelper/Info.plist]: Is this getting hard reset to `2`?

Comment (frederoni) [reply TestHelper/Info.plist]: yep, when agvtool starts incrementing.

Comment (vincethecoder) [reply TestHelper/Info.plist]: @frederoni that doesn't look like an `agvtool` variable. It looks like `$(CURRENT_PROJECT_VERSION)` is replaced with a constant value.

Comment (frederoni) [reply TestHelper/Info.plist]: `agvtool next-version -all` increments the value of `CFBundleVersion` which previously wasn't incrementable so it got bumped to 2.

**Pull request merged**